### ISSUE TITLE
V. 1.2.1 Crash at Startup

### ISSUE THREAD

**Issue by smop95** [NerdHubMC/Cardinal-Components-API#8]
Crash Report: [crash-2019-08-06_00.43.17-client.txt](https://github.com/NerdHubMC/Refined-Machinery/files/3470929/crash-2019-08-06_00.43.17-client.txt)
Log: [2019-08-06-1.log.gz](https://github.com/NerdHubMC/Refined-Machinery/files/3470930/2019-08-06-1.log.gz)

**Comment by UpcraftLP** [comment]
does this also happen with only Refined Machinery installed?

**Comment by UpcraftLP** [comment]
updated crash report: https://paste.ee/p/ficAH.
Incompatibility with OptiFabric

**Comment by UpcraftLP** [comment]
see https://github.com/modmuss50/OptiFabric/issues/55

**Comment by Pyrofab** [comment]
It appears this is in fact an incompatibility with an outdated version of Roughly Enough Items. I would recommend updating to latest version and see if the crash is fixed.

**Comment by Snowfalldevice** [comment]
No even with REI and taking REI out still crashes

https://paste.ee/p/wGyTL

**Comment by NinjaPhenix** [comment]
Just got another report of this issue on expanded storage curseforge page, https://www.curseforge.com/minecraft/mc-mods/expanded-storage-fabric#c178, they also provided crash report which seem to match.

Crash: https://pastebin.com/QEEgkv3w
Log: https://pastebin.com/SZxuDz33

**Comment by UpcraftLP** [comment]
Yup, nothing we can do about that at the moment. Fix is to remove optifine.
see <https://github.com/modmuss50/OptiFabric/issues/55> for updates.
